test/lit/passes/optimize-instructions-gc-tnh-nn.wast

```diff
@@ -1,18 +1,16 @@
 ;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
-;; RUN: wasm-opt %s --optimize-instructions --traps-never-happen --enable-gc-nn-locals -all -S -o - \
+;; RUN: wasm-opt %s --optimize-instructions --traps-never-happen -all -S -o - \
 ;; RUN:   | filecheck %s
 
 (module
   ;; CHECK:      (func $set-of-as-non-null (type $0)
   ;; CHECK-NEXT:  (local $x anyref)
   ;; CHECK-NEXT:  (local.set $x
-  ;; CHECK-NEXT:   (ref.as_non_null
-  ;; CHECK-NEXT:    (local.get $x)
-  ;; CHECK-NEXT:   )
+  ;; CHECK-NEXT:   (local.get $x)
   ;; CHECK-NEXT:  )
   ;; CHECK-NEXT:  (drop
-  ;; CHECK-NEXT:   (local.tee $x
-  ;; CHECK-NEXT:    (ref.as_non_null
+  ;; CHECK-NEXT:   (ref.as_non_null
+  ;; CHECK-NEXT:    (local.tee $x
   ;; CHECK-NEXT:     (local.get $x)
   ;; CHECK-NEXT:    )
   ;; CHECK-NEXT:   )
```

Comment:
Comment on line 21 looks like it needs updating.

Reply:
Looks like I had accidentally changed the behavior of this in OptimizeInstructions. I fixed that, then updated all the comments on these tests.